### DISCUSSION TITLE
Call for topic contributions for the WP Briefing podcast

### DISCUSSION THREAD

**Discussion by nalininsbs** [WordPress/Marketing-Team#53] [Ideas]
There is an open call for topic ideas for the WP Briefing podcast. If there are any topics you would like to hear on the podcast, please contact Santana Inniss on Slack or in this GitHub discussion card.

[WP Briefing Podcast](https://wordpress.org/news/podcast/)

**Comment by iqbalpb** [comment]
Future of Full Site Editing

**Comment by eidolonnight** [comment]
The podcast is on hiatus for now.
